util/formatChartTimes.ts

```diff
@@ -0,0 +1,53 @@
+import { NumberValue } from 'd3'
+
+const createTimeFormatter = (timestamp: NumberValue, locale: string, options: Intl.DateTimeFormatOptions) =>
+  new Date(timestamp.valueOf() * 1000).toLocaleTimeString(locale, options)
+
+export const hourFormatter = (locale: string) => (timestamp: NumberValue) =>
```

Comment:
nit: normally, they are called `timeFormatter` and maybe, because this is a factory method, could be `createTimeFormatter` 

Same for the other, its more common `createDateTimeFormatter`